dotcom-rendering/src/web/components/FixedLargeSlowXIV.tsx

```diff
@@ -95,61 +87,99 @@ export const FixedLargeSlowXIV = ({
-									percentage="25%"
-									showDivider={cardIndex !== 0}
-								>
-									<Card
-										containerPalette={containerPalette}
-										showAge={showAge}
-										linkTo={card.url}
-										format={card.format}
-										headlineText={card.headline}
-										headlineSize="small"
-										imageUrl={
-											groupIndex === 0
-												? card.image
-												: undefined
-										}
-										imagePosition="top"
-										byline={card.byline}
-										showByline={card.showByline}
-										showQuotes={
-											card.format.design ===
-												ArticleDesign.Comment ||
-											card.format.design ===
-												ArticleDesign.Letter
-										}
-										webPublicationDate={
-											card.webPublicationDate
-										}
-										kickerText={card.kickerText}
-										showPulsingDot={
-											card.format.design ===
-											ArticleDesign.LiveBlog
-										}
-										showSlash={true}
-										showClock={false}
-										mediaType={card.mediaType}
-										mediaDuration={card.mediaDuration}
-										starRating={card.starRating}
-										branding={card.branding}
-										discussionId={card.discussionId}
-										dataLinkName={card.dataLinkName}
-										snapData={card.snapData}
-									/>
-								</LI>
-							);
-						})}
-					</UL>
-				);
-			})}
+			<UL direction="row" padBottom={true}>
+				{secondSlice.map((card, cardIndex) => {
+					return (
+						<LI
+							padSides={true}
+							percentage="25%"
+							showDivider={cardIndex !== 0}
+							key={card.url}
+							padBottomOnMobile={cardIndex != 3}
+						>
+							<Card
+								containerPalette={containerPalette}
+								showAge={showAge}
+								linkTo={card.url}
+								format={card.format}
+								headlineText={card.headline}
+								headlineSize="small"
+								imageUrl={card.image}
+								imagePosition="top"
+								byline={card.byline}
+								showByline={card.showByline}
+								showQuotes={
+									card.format.design ===
+										ArticleDesign.Comment ||
+									card.format.design === ArticleDesign.Letter
+								}
+								webPublicationDate={card.webPublicationDate}
+								kickerText={card.kickerText}
+								showPulsingDot={
+									card.format.design ===
+									ArticleDesign.LiveBlog
+								}
+								showSlash={true}
+								showClock={false}
+								mediaType={card.mediaType}
+								mediaDuration={card.mediaDuration}
+								starRating={card.starRating}
+								branding={card.branding}
+								discussionId={card.discussionId}
+								dataLinkName={card.dataLinkName}
+								snapData={card.snapData}
+							/>
+						</LI>
+					);
+				})}
+			</UL>
+			<UL direction="row" padBottom={true} wrapCards={true}>
+				{thirdSlice.map((card, cardIndex) => {
+					return (
+						<LI
+							padSides={true}
+							percentage="25%"
+							showDivider={cardIndex % 4 !== 0}
+							padBottom={cardIndex < 4}
+							padBottomOnMobile={cardIndex !== 11}
```

Comment:
```suggestion
							padBottomOnMobile={cardIndex !== thirdSlice.length - 1}
```

I think the 11 should have been 7 but maybe using length here explains intentions better? I'm not strong on this though so if you prefer the more explicit use of 7 I'm down with that too

Reply:
> I think the 11 should have been 7 but maybe using length here explains intentions better? I'm not strong on this though so if you prefer the more explicit use of 7 I'm down with that too

Great call, thanks!